### PR TITLE
Fix bad table rendering

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.13+

Issue: None

Link to docs preview: [Hot plug/unplug commands](https://61776--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-using-the-cli-tools.html#hot-plug-and-hot-unplug-commands_virt-using-the-cli-tools)

QE review: NA
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-virtctl-commands.adoc [hunk 1]
```diff
@@ -228,20 +228,20 @@ This command removes the association between the VM and the PVC, so that the mem
...
 You use `virtctl` to add or remove resources from running VMs and VMIs.
 
-.Hot plug and hot-unplug commands
+.Hot plug and hot unplug commands
 [width="100%",cols="1a,2a",options="header"]
 |===
 |Command |Description
 |`virtctl addvolume <vm_name> --volume-name=<datavolume_or_PVC> [--persist] [--serial=<label>]`
-|Hot plug a virtual disk by attaching a data volume or persistent volume claim (PVC) to a running VM.
+|Hot plug a data volume by attaching a data volume or persistent volume claim (PVC) to a running VM.
```
Comment (GroceryBoyJr) [inline modules/virt-virtctl-commands.adoc:236]:
```suggestion
|Hot plug a data volume or persistent volume claim (PVC) to a running VM.
```

Comment (GroceryBoyJr) [reply modules/virt-virtctl-commands.adoc:236]: @apinnick I'm treating the term "hot plug" as redundant to the term "attaching".

**Review by GroceryBoyJr — COMMENTED** [review]
Only one small nit!

File: modules/virt-virtctl-commands.adoc (hunk 1 above)
Comment (apinnick) [reply modules/virt-virtctl-commands.adoc:236]: LGTM

**Pull request merged**